src/features/transforms/Transforms.js

```diff
@@ -144,7 +144,7 @@ class Transforms extends Component {
         </Accordion>
       }
 
-      <Card className="mt-3">
+      {!this.props.layer.effect && <Card className="mt-3">
```

Comment:
This isn't how we handle disabling other features (like the transforms).

Reply:
Not sure I understand this...